### ISSUE TITLE
WMS doesn't display along with new basemap (Staman)

### ISSUE THREAD

**Issue by jysuh** [ckan/ckanext-geoview#47]
WMS layer displayed as broken tile on demo.ckan.org site . This is happening on kml layer too. 

![image](https://cloud.githubusercontent.com/assets/20560192/17602021/08602a3a-5fc8-11e6-87e2-988933c42b36.png)

@Corey Erickson

**Comment by jrods** [comment]
@jysuh ran into this problem recently as well (with bcgov map site too ) , try dropping the `request=getcapabilities` parameter in your resource link. I think the map server is having an issue with that parameter.

https://github.com/bcgov/ckanext-bcgov/issues/138

**Comment by jysuh** [comment]
Yes, Indeed. It shows up fine now but somehow, it still shows unknown tiles though. I think it is coming from basemap. Here is snapshot all the layers turned on.
![image](https://cloud.githubusercontent.com/assets/20560192/17607978/9c6db012-5fe6-11e6-8cb1-9e6077f95aeb.png)

Here is snapshot without basemap turned on.

![image](https://cloud.githubusercontent.com/assets/20560192/17607995/b1247b1c-5fe6-11e6-851e-eb7535d710c1.png)

@Corey Erickson

**Comment by mattfullerton** [comment]
@jysuh Try replacing the {s} in the base map path in the config with a "b" or just removing it completely. I seem to remember that the OpenLayers-based views in GeoView don't do the subdomains part properly. For which we probably should have an issue somewhere.
